### ISSUE TITLE
Poisson regression, pretty please :-)

### ISSUE THREAD

**Issue by fusaroli** [LudvigOlsen/R-cross_validate#6]
Probably second to last issue (I'll be trying tomorrow the batch function). What about including Poisson regression (outcome is a count variable)? That's the third most common form of regression and it's relatively easy to implement: it works just like the linear regression. The main difference is that estimates have to be exponentiated to be interpreted (exp(x)), but that's of no relevance to the cross-validation. Performance is still measured as rmse. An example of poisson tutorial (not using glmer): http://www.ats.ucla.edu/stat/r/dae/poissonreg.htm

**Comment by LudvigOlsen** [comment]
Yes, I have given this some thought, and if it is easy, it will probably be fast to implement!
